packages/app-headless-cms/src/admin/views/EnvironmentAliases/EnvironmentAliases.tsx

```diff
@@ -21,15 +21,18 @@ function EnvironmentAliases() {
                 mutation: DELETE_ENVIRONMENT_ALIAS,
                 options: {
                     refetchQueries: [
-                        { query: LIST_ENVIRONMENTS, variables: { sort: { savedOn: -1 } } }
+                        { query: LIST_ENVIRONMENTS, variables: { sort: { savedOn: -1 } } },
                     ]
                 }
             }}
             read={READ_ENVIRONMENT_ALIAS}
             create={{
                 mutation: CREATE_ENVIRONMENT_ALIAS,
                 options: {
-                    update: updateCacheAfterCreate
+                    update: updateCacheAfterCreate,
+                    refetchQueries: [
```

Comment:
Since we're already doing it for updating environments, let's also update the cache with necessary changes in the `updateCacheAfterCreate`, instead of making a separate call to API.